content/docs/security/hipaa.md

```diff
@@ -66,6 +66,129 @@ HIPAA is a federal law that sets national standards for the protection of health
 
 Audit events may not be logged if database endpoints experience exceptionally heavy load, as we prioritize database availability over capturing log events.
 
+## Logged events
+
+Neon maintains a comprehensive audit trail to support HIPAA compliance. This includes two categories of logged events:
+
+1. **SQL activity**, logged using the [pgAudit](https://www.pgaudit.org/) extension (`pgaudit`) for PostgreSQL.
+2. **User and administrative actions** performed via the Neon Console, logged at the API request level.
+
+For more on pgAudit, see the [pgAudit documentation](https://github.com/pgaudit/pgaudit/blob/main/README.md).
+
+### pgAudit settings in Neon (HIPAA mode)
+
+When HIPAA audit logging is enabled for a Neon project, Neon configures pgAudit with the following settings by default:
+
+| Setting                      | Value        | Description                                                                                   |
+| ---------------------------- | ------------ | --------------------------------------------------------------------------------------------- |
+| `pgaudit.log`                | `all, -misc` | Logs all classes of SQL statements except low-risk miscellaneous commands.                    |
+| `pgaudit.log_parameter`      | `off`        | Parameters passed to SQL statements are not logged to avoid capturing sensitive values.       |
+| `pgaudit.log_catalog`        | `off`        | Queries on system catalog tables (e.g., `pg_catalog`) are excluded from logs to reduce noise. |
+| `pgaudit.log_statement`      | `on`         | The full SQL statement text is included in the log.                                           |
+| `pgaudit.log_relation`       | `off`        | Only a single log entry is generated per statement, not per table or view.                    |
+| `pgaudit.log_statement_once` | `off`        | SQL statements are logged with every entry, not just once per session.                        |
+
+#### What does `pgaudit.log = 'all, -misc'` include?
```

Comment:
We should include a note that explains the risk identified in https://github.com/neondatabase/cloud/issues/28839 - basically, we depend on pgaudit to correctly redact passwords, but there is always a possibility that credentials may be exposed in audit logs.  We have no way of preventing this / known risk that we're calling out.